server/App.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+const cors = require('cors');
+const bodyParser = require('body-parser');
+const httpServer = require('http').Server;
+const { whoIsIt } = require('./face-recognitor/face-recognitor');
+const { saveImage, removeImage} = require('./utils/image-utils');
+
+const PORT = process.env.PORT || 3000
+const app = express()
+const server = httpServer(app)
+
+app.use(cors())
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({ extended: false, limit: '10mb' }))
+
+app.get('/', async (req, res) => {
+  res.send('Server is running');
+});
+
+app.post('/whoisit', async (req, res) => {
+    console.log('POST /api/whoisit')
+
+    const { image } = req.body
+    if (!image) return res.sendStatus(400)
+
+    const imagePath = await saveImage(new Buffer(image.split(',')[1], 'base64'))
```

Comment:
If this promise rejects, the route function will throw an unhandled rejection error. I'd suggest wrapping the whole thing in a try-catch block, or even better, wrapping the async route function with a wrapper function that handles promise rejections (see smashdown backend for example).

Also, it's good to add a global error handler to the server to e.g. return 500 with some logging if something unexpected happends.